### ISSUE TITLE
Add layer filesystem changes to exported json

### ISSUE THREAD

**Issue by RaananHadar** [wagoodman/dive#333]
One of the best features of dive is showing which files changed between the layers in the tree view. However this data is currently not exported when using -j. 

Inspired by [this post](https://appfleet.com/blog/reverse-engineer-docker-images-into-dockerfiles-with-dedockify/), I really wish for tools that will allow to reproduce docker files. This is very important for inspecting docker images for on prem environments in terms of security. For this a user will copy the changed files in the image outside of the container as a first step. This can be achieved today with dive, only via the tree view, this is manual and inefficient for complex images and has a huge potential for automation. 

In short, it would be really useful if for each layer in the json, the paths of the modified files, added files and removed files can be exported as well.

**Comment by rulatir** [comment]
YESSSSS to this!

**Comment by mikesir87** [comment]
I would love to see this as well. I'm not sure what other tools might be consuming this right now, but I'd anticipate the sudden addition of the layer filesystem changes could be quite large, depending on the size of the image. I'd be open to the idea of yet another flag to include these extra pieces as well.

My use case is looking at providing a browser-based UI and help folks see how to better their images.

**Comment by Akash-Nayak** [comment]
PR https://github.com/wagoodman/dive/pull/458 is for adding this feature to the Dive tool.
